### PR TITLE
API hw 1 done

### PR DESCRIPTION
**Pull request by PIDAMI**
details in readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
-# HW1
+# API homework 1
 
-Calculator jar should be located in src/test/resources folder
+My simple collection, consisting of CRUD operations 
+for Trello's boards, lists and cards can be found [here]( https://go.postman.co/workspace/My-Workspace~2e0dfda3-9ddf-4566-848f-62c8b532d67d/collection/17049177-e87f6a17-01ba-4b1c-b0f9-28b85c04d61a?action=share&creator=17049177)
```
Comment (alla-blinkova) [inline README.md]: I don't have access to the workspace. Could you add all the files to the pull request? It will be easier to review

**Comment by alla-blinkova** [comment]
Why files from one of the previous homeworks have changed?

**Review by alla-blinkova — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PIDAMI) [reply README.md]: added collection as json

**Review by alla-blinkova — CHANGES_REQUESTED** [review]
Credentials should not be stored in repository, please delete key and token from the history

**Pull request closed without merging**